util/exponential-fallback.js

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+const MAX_RETRIES = 5
+const MAX_WAIT_INTERVAL_MS = 1 * 60 * 1000 // 1 minute in milliseconds
```

Comment:
1 minute is way too long, we have to consider the use cases where we access the tvm: files and states initialization are on the critical request path. 
+ I would remove those vars and pass them as params to the function
+ reasonable defaults to me would be **{ retries: 5, multiplier: 1.5, interval: 500ms }**